src/Lib/AutoLoginTrait.php

```diff
@@ -109,6 +110,6 @@ public function getKey(): string
      */
     public function getSalt(): string
     {
-        return $this->user_salt;
+        return Configure::read('auto_login.salt');
```

Comment:
Probably won't work, as the auth_actions config is not loaded by default, see `AuthActionsTrait` where it is always loaded first.

It normally is not a globally loaded config anyway, I'm not sure whether it is such a good idea to make it one now or just stick with the pattern and load it on demand.